### PR TITLE
Add information about the clip module

### PR DESCRIPTION
**Pull request by bnmajor**
Signed-off-by: Brianna Major <brianna.major@kitware.com>

### CODE REVIEW

File: docs/visualization.md [hunk 1]
```diff
@@ -128,6 +128,25 @@ panel shown.
 
 ![Threshold](img/threshold2.png)
 
+### Clip
+
+Clipping planes can be added to the data in order to clip any applied `Slice`,
+`Volume`, or `Contour` modules. The default is an orthogonal plane, but the its
```
Comment (cryos) [inline docs/visualization.md:134]: s/the its/its

File: docs/visualization.md [hunk 2]
```diff
@@ -128,6 +128,25 @@ panel shown.
...
+direction can be changed and the plane can be inverted to allow for clipping
+from all directions.
```
Comment (cryos) [inline docs/visualization.md:136]: s/all/any

File: docs/visualization.md (hunk 1 above)
Comment (bnmajor) [reply docs/visualization.md]: Changed

File: docs/visualization.md (hunk 2 above)
Comment (bnmajor) [reply docs/visualization.md]: Changed

**Pull request merged**